build/configs/artik053/extra/defconfig

```diff
@@ -1139,6 +1138,12 @@ CONFIG_TASH_CMDTASK_PRIORITY=100
 # CONFIG_SYSTEM_FOTA_HAL is not set
 # CONFIG_SYSTEM_I2CTOOL is not set
 # CONFIG_SYSTEM_INIFILE is not set
+CONFIG_SYSTEM_PREAPP_INIT=y
+CONFIG_SYSTEM_PREAPP_STACKSIZE=2048
+
+#
+# FLASH Program Installation
```

Comment:
No. I mean "FLASH Program Installation". I can not find comment "FLASH Program Installation" in Kconfig. It means "did you modify this manually?"